### PR TITLE
Issue #20 Fix RAM size in fabric8 add-on usage guide

### LINKED ISSUE #20: Fix RAM size in fabric8 add-on usage guide
The RAM size as mentioned in [fabric8 usage guide](https://github.com/minishift/minishift-addons/tree/master/add-ons/fabric8#start-minishift) is 5 GB and the command below
```
$ minishift start --memory=5000 --cpus=5 --disk-size=50g
```

uses 5000 which get printed as 4GB as per new pre-flight startup logs.
```
$ minishift start --memory 5000 --cpus=5 --disk-size=40g
-- Installing default add-ons ... OK
-- Checking if KVM driver is installed ... 
   Driver is available at /usr/local/bin/docker-machine-driver-kvm ... OK
-- Starting local OpenShift cluster using 'kvm' hypervisor ...
-- Minishift VM will be configured with ...
   Memory:    4 GB
   vCPUs :    5
   Disk size: 40 GB
```

### PR DESCRIPTION
**Pull request by budhrg**
Fix #20

### CODE REVIEW

**Comment by budhrg** [comment]
@hferentschik Small fixes.

File: add-ons/fabric8/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ We currently recommend you use at least 5Gb of RAM in your minishift VM and toda
 
 So start minishift using something like this:
 
-    $ minishift start --memory=5000 --cpus=5 --disk-size=50g
+    $ minishift start --memory=5120 --cpus=5 --disk-size=50g
```
Comment (hferentschik) [inline add-ons/fabric8/README.md]: I don't think this looks nice. Does 5G or 5GB not work? If not let's wait until https://github.com/minishift/minishift/issues/1068 is resolved.

File: add-ons/fabric8/README.md [hunk 2]
```diff
@@ -62,5 +62,5 @@ where the above `TODO` text is replaced by the actual client id and secret from
 
 Once you have the 2 GitHub environment variables defined you can apply the Fabric8 addon via:
 
-    $ minishift addon apply --addon-env GITHUB_OAUTH_CLIENT_ID=$GITHUB_OAUTH_CLIENT_ID --addon-env GITHUB_OAUTH_CLIENT_SECRET=$GITHUB_OAUTH_CLIENT_SECRET fabric8
-
+    $ minishift addon apply --addon-env GITHUB_OAUTH_CLIENT_ID=$GITHUB_OAUTH_CLIENT_ID --addon-env
+      GITHUB_OAUTH_CLIENT_SECRET=$GITHUB_OAUTH_CLIENT_SECRET fabric8
```
Comment (hferentschik) [inline add-ons/fabric8/README.md]: -1, this is suppused to be on a single line

File: add-ons/fabric8/README.md (hunk 1 above)
Comment (gbraad) [inline add-ons/fabric8/README.md]: I strongly suggest you to use a unit instead of a bare number

Comment (budhrg) [inline add-ons/fabric8/README.md]: :+1:  Sure will wait for https://github.com/minishift/minishift/issues/1068 to get resolved then we can add 5GB which will look more intuitive.

Comment (gbraad) [inline add-ons/fabric8/README.md]: Memory will be done as #1149

Comment (gbraad) [reply add-ons/fabric8/README.md]: RAM fix has been merged into Minishift and released as 1.5.0

**Comment by budhrg** [comment]
@gbraad Done.

**Comment by hferentschik** [comment]
Merged, thanks.

File: add-ons/fabric8/README.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ We currently recommend you use at least 5Gb of RAM in your minishift VM and toda
 
 So start minishift using something like this:
 
-    $ minishift start --memory=5000 --cpus=5 --disk-size=50g
+    $ minishift start --memory=5GB --cpus=5 --disk-size=50g
```
Comment (gbraad) [inline add-ons/fabric8/README.md]: I hope you also changed it for the disk-size

**Pull request merged**